src/main/webapp/app/config/constants.tsx

```diff
@@ -534,7 +534,6 @@ export enum PAGE_ROUTE {
   LOGIN = '/login',
   LOGOUT = '/logout',
   LEGACY_DATA_ACCESS = '/dataAccess',
-  DATA_ACCESS = '/data-access',
```

Comment:
We no longer have data access page. Previously we wanted to redirect to from /dataAccess to /api-access. This was added by mistake.